### PR TITLE
JFG-874 Add checkstyle to Jaas

### PR DESCRIPTION
**Pull request by anton-antonenko**
JFG-874: Add checkstyle check during build for Jaas;
fix checkstyle issue in JaasRestController;
add checkstyle-checker.xml file for maven/idea/eclipse/etc  checkstyle plugins.

### CODE REVIEW

**Comment by SokolAndrey** [comment]
+1

File: jaas/pom.xml [hunk 1]
```diff
@@ -197,6 +200,43 @@
...
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>findbugs-maven-plugin</artifactId>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-pmd-plugin</artifactId>
+            </plugin>
+            <plugin>
```
Comment (dlatnikov) [inline jaas/pom.xml]: why is the check style plugin only in jaas?  what about dbapi?

Comment (anton-antonenko) [inline jaas/pom.xml]: There're was nothing said about dbapi in the ticket.
I'll add it to dbapi also, but without build failing

**Comment by anton-antonenko** [comment]
@a-badaev follow these steps: https://youtrack.jetbrains.com/issue/IDEA-61520#comment=27-1292600

**Pull request merged**